test/plugin/scenarios/armeria-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,141 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+registryItems:
+  applications:
+    - {armeria-server: 2}
+  instances:
+    - {armeria-server: 1}
+  operationNames:
+    - armeria-server: [/greet/skywalking, /healthCheck]
```

Comment:
Health check should not be included in the test expected data.

Reply:
Almost all the other test scenarios include `healthCheck`, and removing this causes failure, I don't think we should remove it. @arugal is it the case?

Reply:
> and removing this causes failure, I don't think we should remove it.

Removing does not failure, in the latest validator rule.

Reply:
> > and removing this causes failure, I don't think we should remove it.
> 
> Removing does not failure, in the latest validator rule.

well, I tested it with older codes, now it passed